index.html

```diff
@@ -1,75 +1,105 @@
-        <section class="top-content">
-            <div class="text-container">
-                <h2>The Future</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-            <div class="text-container">
-                <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-        </section>
-        
-        <section class="middle-content">
-        
-            <h2>Why Did It Have To Be Boxes...</h2>
-            
-            <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
-        
-        </section>
+      <header>
+        <nav class="navigation">
+          <img class="logo" src="img/lambda-black.png" alt="Lambda logo" />
+          <a href="index.html">Home</a>
+          <a href="about.html">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>
+        </nav>
+
+        <img class="jumbo-img" src="img/jumbo.jpg" alt="header image" />
+      </header>
+
+      <section class="top-content">
+        <div class="text-container">
+          <h2>The Future</h2>
+          <p>
+            Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+            Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+        <div class="text-container">
+          <h2>The Past</h2>
+          <p>
+            Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+            Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+      </section>
+
+      <section class="middle-content">
+        <h2>Why Did It Have To Be Boxes...</h2>
 
-        <section class="bottom-content">
+        <div class="boxes">
+          <div class="box teal">Box 1</div>
```

Comment:
I liked how you named the class the color of what the box should be that way you can easily style it. I named mine box1 box2 but the way you did it makes it easier.